adoc/Private-Registry-Administration.adoc

```diff
@@ -110,35 +110,251 @@ $ uaac client add suse_private_registry -s suseprivateregistrysupersecret --scop
+* replacing the database service
+
+Helm configuration changes can usually be applied by updating the {spr} `harbor-values.yaml` configuration file used during installation with the new configuration values and then running `helm upgrade` to apply the changes. Cases that required additional steps are explicitly documented in the sub-sections that follow.
+
+[TIP]
+====
+The full list of helm configuration options and default values that can overridden in the `harbor-values.yaml` file is included in the helm chart itself and can be viewed in YAML format by running the following command:
+
+[source,bash]
+----
+helm show values harbor
+----
+
+The {spr} Deployment section also contains extended information on the most relevant helm configuration options. Those configuration options can be customized not only during installation but also updated post-installation, with the exceptions documented earlier in this section as not supported.
+====
 
-// === Changing or Resetting Passwords
-//
-// ==== Changing the Password for Internal Database
-// // TODO AI (Stefan Nica ) needs content
-//
-// ==== Changing the Password for External Database
-// // TODO AI (Stefan Nica ) needs content
-//
-// ==== Changing the Password for Internal Redis
-// // TODO AI (Stefan Nica ) needs content
-//
-// ==== Changing the Password for External Redis
-// // TODO AI (Stefan Nica ) needs content
-//
-// === Rotating TLS Certificates
-// // TODO AI (Stefan Nica ) needs content
+For example, to enable internal TLS, notary and trivy in one go (assuming they are all currently disabled), update the `harbor-values.yaml` configuration file and run `helm upgrade` as follows:
 
+.harbor-values.yaml
+[source,yaml]
+----
+internalTLS:
+  enabled: true
+trivy:
+  enabled: true
+  replicas: 3
+  gitHubToken: "<github-auth-token>"
+notary:
+  enabled: true
+  server:
+    replicas: 3
+  signer:
+    replicas: 3
+----
+
+[source,bash]
+----
+helm -n registry upgrade suse-registry ./harbor -f harbor-values.yaml
+----
+
+==== Changing the Database Password
+[CAUTION]
+====
+Changing the database password is an administrative operation that has impact on service availability.
+====
+
+The password for the {spr} internal or external database service can be changed in three steps:
+
+. first, shutdown all {spr} services that use the database, to eliminate the risk of incomplete or failed transactions:
++
+[source,bash]
+----
+kubectl -n registry scale deployment -l component=core --replicas=0
+kubectl -n registry scale deployment -l component=notary-server --replicas=0
+kubectl -n registry scale deployment -l component=notary-signer --replicas=0
+----
+
+. change the password for the database server
++
+For an external database, use the means available from the public cloud provider to set a new admin password.
++
+For the internal database, this can only be done by accessing the database pod via `kubectl exec` and running a `psql` command to change the password, e.g.:
```

Comment:
That's not the _only_ way. You can e.g. use `kubectl port-forward` to forward the port from a service to your local machine and than access localhost with `psql`...

Reply:
@jsuchome that's true, but psql is already installed in the container, which makes it more convenient :)

Reply:
Sure, you do not have to cover all options, just do not mention the _only way_ is using `kubectl exec`